static/css/style.css

```diff
@@ -185,7 +190,13 @@ ul {
 }
 
 .breadcrumb {
-  width: 100%;
+  width: fit-content;
+  border-radius: 8px;
+  padding: 4px 8px;
+  background-color: var(--green-aquamarine);
```

Comment:
Use the navbar color and put text white with no line decoration for link (analytics page)